src/DynamoCoreWpf/Views/Core/AnnotationView.xaml

```diff
@@ -724,7 +724,6 @@
                               Grid.Row="0"
                               Canvas.ZIndex="20"
                               HorizontalContentAlignment="Left"
-                              Style="{StaticResource InOutPortControlStyle}"
```

Comment:
what consequences does removing these styles have?

Reply:
This styling doesn't make any additional difference to other ports as this same style will be inherited from the node ports(they have this same styling)